### ISSUE TITLE
Sort "Badge Holders" names alphabetically

### ISSUE THREAD

**Issue by stickster** [fedora-infra/tahrir#382]
Badge pages have a list of users who were awarded the badge. As someone who awards badges, this list is hard to parse for a previous awardee since it's not ordered. Ideally the list should be alphanumerically sorted so I can locate who's already been awarded the badge.

**Comment by muznahaftab** [comment]
Hi @stickster I'm an outreachy applicant. May I work on this issue?

**Comment by justwheel** [comment]
Hi @muznahaftab! If you are interested in working on the Badges project as part of Outreachy, please take a look at this page and start here. :smile: 

https://docs.fedoraproject.org/en-US/badges/join/

**Comment by whybepb** [comment]
Hey can i work on this issue

**Comment by penguinpee** [comment]
Sure you can, @Bprathmesh. Feel free to submit a PR mentioning this issue.

**Comment by abompard** [comment]
Hey @Bprathmesh , are you still interested in working on this? Or should I unassign you?

**Comment by olamidepeterojo** [comment]
Hello @abompard, I am an Outreachy applicant for this year's June cohort and also participated in the contribution phase of the June 2024 cohort. I'd like to pick up this issue to get used to the code base

**Comment by gridhead** [comment]
This suggestion does not make sense anymore, as we want the badge owners' names to be sorted according to the order in which the owners received them, so of course, someone who got the badge first should be celebrated because they were able to complete the required action and not simply because their name begins with `A`. I am closing this as unplanned.

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/54d14905-ac9e-49d7-bd2b-367e0daf193f" />
